development.txt

```diff
@@ -136,5 +163,7 @@ Save and exit
 make -j$(nproc)
 --------------------------------------------------------------------------------
 
+Where you should replace +$(nproc)+ with the number of _physical_ CPU cores obtainable from +lscpu+ command as _Core(s) per socket_.
```

Comment:
`$(nproc)` return the number of cores presents on the machine no need to do it manually

Reply:
$(nproc) considers all the CPUs also due to hyperthreading, in my experience (with computational stuff just using CPU+RAM and nothing else, not with compilations, which could be different as it uses CPU+RAM+Disk+Network) is noticeably **faster** to use just as many threads as the **physical** cores.
I just tested on my laptop with a Intel i7-4600U @ 2.10GHz CPU (Thread(s) per core: 2, Core(s) per socket: 2) using buildroot and doing `make clean` + `make -j#`:
using 4 threads it took 14 min 22 s
using 2 threads it took 19 min 45 s
ok, so seems that with the kind of processes involved with OpenWrt compilation, the more processes the better, in the new PR I will remove this comment.